src/components/00-materials/README.md

```diff
@@ -99,6 +99,10 @@ body {
 
 ## Icon/Image set
 
+### Size classes
```

Comment:
I would prefer "Sizes" over "Size classes". I know what you mean but I think it is confusing.